practice_07.13.2019_events/Task2_HunterAndRabbit/Hunting.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace practice_07._13._2019_events.Task2_HunterAndRabbit
+{
+    public class Hunting
+    {
+        public Hunter Hunter { get; set; }
+
+        public Rabbit Rabbit { get; set; }
+
+        public Hunting(Hunter hunter, Rabbit rabbit)
+        {
+            this.Hunter = hunter;
+            this.Rabbit = rabbit;
+
+            this.Rabbit.EventNewLocation += this.OnEventNewLocationInvoked;
```

Comment:
Please remember that we should unsubscribe from events.

Reply:
As I understood correctly, the following row should be in the end of the ctor?:
this.Rabbit.EventNewLocation -= this.OnEventNewLocationInvoked